package.nls.json

```diff
@@ -160,6 +160,7 @@
     "AWS.command.quickStart.error": "There was an error retrieving the Quick Start page",
     "AWS.command.s3.downloadFileAs": "Download As...",
     "AWS.command.s3.copyPath": "Copy Path",
+    "AWS.command.s3.addBucket": "Add Existing Bucket...",
```

Comment:
should it be "Show" instead of "Add", for parallel form with `Show region` action?